Resources/Prototypes/_CM14/Entities/Objects/Medical/healing.yml

```
@@ -0,0 +1,98 @@
+- type: entity
+  parent: BaseItem
+  id: CMBaseHealingItem
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: _CM14/Objects/Medical/medical.rsi
+  - type: Item
+    size: Small
+    sprite: _CM14/Objects//Medical/medical.rsi
+    heldPrefix: ointment
+  # Inherited
+  - type: StaticPrice
+    price: 0
+
+- type: entity
+  name: ointment
+  description: Used to treat those nasty burns. Less effective on caustic burns.
+  parent: CMBaseHealingItem
+  id: CMOintment  
+  suffix: Full
+  components:
+  - type: Tag
+    tags:
+    - CMOintment
+  - type: Sprite
+    state: ointment
+  - type: Item
+    heldPrefix: ointment
+  - type: Healing
+    damageContainers:
+      - Biological
+    damage:
+      types:
+        Heat: -5
+        Cold: -5
+        Shock: -5
+        Caustic: -1.5
+    healingBeginSound:
+      path: "/Audio/Items/Medical/ointment_begin.ogg"
+    healingEndSound:
+      path: "/Audio/Items/Medical/ointment_end.ogg"
+  - type: Stack
+    stackType: CMOintment
+    count: 10
+  - type: StackPrice
+    price: 10
+
+- type: entity
+  name: medical gauze
+  description: Some sterile gauze to wrap around bloody stumps and lacerations.
+  parent: CMBaseHealingItem
+  id: CMBrutepack
+  suffix: Full
+  components:
+  - type: Tag
+    tags:
+    - Brutepack
+  - type: Sprite
+    state: brutepack
+  - type: Healing
+    damageContainers:
+      - Biological
+    damage:
+      groups:
+        Brute: -15
+        Slash: -5
+        Piercing: -10
```

`Slash` and `Piercing` should be under `types`, not `groups`

```suggestion
    damage:
      groups:
        Brute: -15
      types:
        Slash: -5
        Piercing: -10
```